ros/src/twist_controller/twist_controller.py

```diff
@@ -66,7 +66,7 @@ def control(self, current_vel, dbw_enabled, linear_vel, angular_vel):
 
         if linear_vel == 0. and current_vel < 0.1:
             throttle = 0
-            brake = 400 #N*m - to hold the car in place if we are stopped at a light. Acceleration ~ 1m/s^2
+            brake = 700 #N*m - to hold the car in place if we are stopped at a light. Acceleration ~ 1m/s^2
```

Comment:
could you add your assumption here? Why 700 is better than 400? This will be necessary for other people will understand this later.